GVRf/Framework/src/org/gearvrf/GVRLight.java

```diff
@@ -29,8 +29,49 @@ public GVRLight(GVRContext gvrContext) {
         setDiffuseIntensity(1.0f, 1.0f, 1.0f, 1.0f);
         setSpecularIntensity(1.0f, 1.0f, 1.0f, 1.0f);
         isEnabled = true;
+        name="";       
```

Comment:
Or ... you could use the Java formatter recommended on the Gear VRf website:

http://gearvrf.org/bin/view/GearVRF/GearVRfWikiGetInvolved#SubmitPatch

Look under the "Coding Guidelines" header.